### ISSUE TITLE
"Feature `generators` has been removed" error with rustc 1.75 nightly

### ISSUE THREAD

**Issue by beeb** [deeprobin/bruteforce-rs#28]
When trying to compile the library with the latest nightly, I get the following error:

```
   Compiling bruteforce v0.2.1
error[E0557]: feature has been removed
 --> /home/.../bruteforce-0.2.1/src/lib.rs:4:18
  |
4 | #![feature(test, generators, proc_macro_hygiene)]
  |                  ^^^^^^^^^^ feature has been removed
  |
  = note: renamed to `coroutine`
```

**Comment by deeprobin** [comment]
Thank you very much.

I can look for it the days.
@beeb Or do you want to submit a PR? - it's Hacktober

**Comment by beeb** [comment]
I tried briefly because I want to use this library right now, but hit other errors and finally simply removed the generators stuff from my fork to be able to keep working. I also had errors with the `charset!` macro. Not sure I am able to fix it myself.

**Comment by deeprobin** [comment]
Good to know. I'll try to work on this tomorrow or the day after, depending on my schedule ...

**Comment by beeb** [comment]
Thanks! No rush for me as I found a workaround by forking it :)

**Comment by deeprobin** [comment]
#29 fixed it. Change available in version `0.3.0`.
Thank you @Nereuxofficial for your effort.
